rest/route/service.go

```diff
@@ -92,6 +92,8 @@ func AttachHandler(app *gimlet.APIApp, opts HandlerOpts) {
 	app.AddRoute("/hosts/{host_id}/logs/error").Version(2).Get().RouteHandler(makeContainerLogsRouteManager(sc, true))
 	app.AddRoute("/hosts/{task_id}/create").Version(2).Post().Wrap(checkTask).RouteHandler(makeHostCreateRouteManager(sc))
 	app.AddRoute("/hosts/{task_id}/list").Version(2).Get().Wrap(checkTask).RouteHandler(makeHostListRouteManager(sc))
+	app.AddRoute("/volumes").Version(2).Post().Wrap(checkUser).RouteHandler(makeCreateVolume(sc))
+	app.AddRoute("/volumes/{volume_id}").Version(2).Wrap(checkUser).Delete().RouteHandler(makeDeleteVolume(sc))
```

Comment:
should we also add a list volumes? 

should these be hosts/volumes methods or no?

Reply:
I don't see a huge reason why they can't be hosts/volumes. Was worried that we would have a POST route on hosts/volumes that would conflict with a POST route like hosts/host_id (which we don't have).

Reply:
Created https://jira.mongodb.org/browse/EVG-6841